test/test421.js

```diff
@@ -57,4 +57,32 @@ describe('Test 421 Test for JOINSTAR', function () {
 		]);
 		done();
 	});
+
+	it('5. JSON JOINSTAR with qualified names', function (done) {
+		alasql.options.joinstar = 'json';
+		alasql.databases.test421.dbversion++; // Reset database cache
+		var res = alasql('SELECT * FROM test' + test + '.one, test' + test + '.two');
+		//console.log(res);
+		assert.deepEqual(res, [
+			{one: {a: 1}, two: {a: 10}},
+			{one: {a: 1}, two: {a: 20}},
+			{one: {a: 2}, two: {a: 10}},
+			{one: {a: 2}, two: {a: 20}},
+		]);
+		done();
```

Comment:
Please only use done() when we have an async test. 

for sync test we just use ()=>{